### PR TITLE
Improve doc for RBAC group task

### PR DESCRIPTION
**Pull request by pitlv2109**

### CODE REVIEW

**Comment by pitlv2109** [comment]
@geeknoid  It's still "Waiting on code owner review from frankbu, geeknoid, rcaballeromx, and/or sdake"

Do you need to add lgtm tag?

File: content/docs/tasks/security/rbac-groups/index.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ JWT authentication for `httpbin`.
     EOF
     {{< /text >}}
 
+    Policy needs some time to take affect. Wait at least one minute.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]:
```suggestion
    Policies take effect after some time, please wait at least one minute.
```

File: content/docs/tasks/security/rbac-groups/index.md [hunk 2]
```diff
@@ -136,6 +138,10 @@ wait until the newly defined RBAC policy to take effect.
 
 1.  Enable the Istio RBAC for the namespace:
 
+    {{< warning >}}
+    This is a cluster-wide policy. If your cluster is using another `ClusterRbacConfig`, it will be overwritten.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]:
```suggestion
    This is a cluster-wide policy and Istio overwrites any another `ClusterRbacConfig`.
```

File: content/docs/tasks/security/rbac-groups/index.md [hunk 3]
```diff
@@ -241,6 +247,11 @@ You may use the `gen-jwt`
 [python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
 to generate a JWT with other list-typed claims for testing purposes.
 Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.
+For the `key.pem` file, you may use one at
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]:
```suggestion
To get the `key.pem` file, run the following command.
```

File: content/docs/tasks/security/rbac-groups/index.md (hunk 1 above)
Comment (pitlv2109) [reply content/docs/tasks/security/rbac-groups/index.md]: Done. Thanks!

File: content/docs/tasks/security/rbac-groups/index.md (hunk 2 above)
Comment (pitlv2109) [reply content/docs/tasks/security/rbac-groups/index.md]: Done. Thanks!

File: content/docs/tasks/security/rbac-groups/index.md (hunk 3 above)
Comment (pitlv2109) [reply content/docs/tasks/security/rbac-groups/index.md]: Done. Thanks!

**Comment by pitlv2109** [comment]
@rcaballeromx @geeknoid @sdake @frankbu Could you review or merge this PR? It's long overdue... :(

File: content/docs/tasks/security/rbac-groups/index.md [hunk 4]
```diff
@@ -153,6 +155,10 @@ wait until the newly defined RBAC policy to take effect.
 
 1.  Enable the Istio RBAC for the namespace:
 
+    {{< warning >}}
+    This is a cluster-wide policy and Istio overwrites any another `ClusterRbacConfig`.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]: Please include the nouns. For example `This` without a noun might lead to readers not knowing what you are referencing. Consider:
```suggestion
    The Istio RBAC enablement is a cluster-wide policy and Istio overwrites any previously set `ClusterRbacConfig` configuration.
```

File: content/docs/tasks/security/rbac-groups/index.md [hunk 5]
```diff
@@ -258,6 +264,11 @@ You may use the `gen-jwt`
 [python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
 to generate a JWT with other list-typed claims for testing purposes.
 Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.
+To get the `key.pem` file, run the following command:
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]: Please make this part of the numbered steps.
```suggestion
1. Get the `key.pem` file with the following command:
```
```suggestion
To get the `key.pem` file, run the following command:
```

File: content/docs/tasks/security/rbac-groups/index.md [hunk 6]
```diff
@@ -258,6 +264,11 @@ You may use the `gen-jwt`
...
+
+{{< text bash >}}
```
Comment (rcaballeromx) [inline content/docs/tasks/security/rbac-groups/index.md]: Please indent this code block with 4 spaces to ensure the numbered list doesn't break.

File: content/docs/tasks/security/rbac-groups/index.md [hunk 7]
```diff
@@ -122,6 +122,8 @@ communicating with `httpbin`.
     EOF
     {{< /text >}}
 
+    Policies take effect after some time, please wait at least one minute.
```
Comment (rlenglet) [inline content/docs/tasks/security/rbac-groups/index.md]: Is there a way to check when the policies have taken effect? If so, document it, and update all the other places in this doc that request the user to wait for policies to take effect.

File: content/docs/tasks/security/rbac-groups/index.md [hunk 8]
```diff
@@ -258,6 +264,11 @@ You may use the `gen-jwt`
 [python script]({{<github_file>}}/security/tools/jwt/samples/gen-jwt.py)
 to generate a JWT with other list-typed claims for testing purposes.
 Follow the instructions in the `gen-jwt` script to use the `gen-jwt.py` file.
+To get the `key.pem` file, run the following command:
+
+{{< text bash >}}
+$ wget {{< github_file >}}/security/tools/jwt/samples/key.pem
```
Comment (rlenglet) [inline content/docs/tasks/security/rbac-groups/index.md]: Replace `wget` with `curl`. The majority of the docs use `curl`:
```
$ grep -r wget content/ | wc -l
       4
$ grep -r curl content/ | wc -l
     263
```

**Comment by sdake** [comment]
@pitlv2109 if you can address reviewer comments and rebase so we can merge this work, I would appreciate it.

Cheers
-steve

**Comment by geeknoid** [comment]
Closing due to inactivity. Please reopen if you have some time to finish the work.

**Pull request closed without merging**